package.json

```
@@ -18,7 +18,7 @@
     "weedout": "~1.0.1"
   },
   "peerDependencies": {
-    "mongoose": "^5.8.10"
+    "mongoose": "^6.2.1"
```

Let's be loose here so we don't bug people with warnings if, for whatever reason, they're on earlier versions:

```suggestion
    "mongoose": "6.x"
```